R/ard_attributes.R

```diff
@@ -1,29 +1,48 @@
-#' ARD Attributes
+#' Add variable attributes to an ARD object
 #'
+#' @description
+#' S3 generic providing methods for the following classes:
+#'
+#' * `data.frame`: Adds variable attributes to an ARD data frame.
+#'
+#' @param data (`data.frame`)\cr
+#'   a data frame
+#' @param ... Arguments passed to other methods
+#'
+#' @return an ARD data frame of class 'card'
+#'
+#' @examples
+#' df <- dplyr::tibble(var1 = letters, var2 = LETTERS)
+#' attr(df$var1, "label") <- "Lowercase Letters"
+#'
+#' ard_attributes(df, variables = everything())
+#'
+#' @export
+#'
+ard_attributes <- function(data, ...) {
```

Comment:
In other generics in the package, we name the primary argument `x` to allow us to expand in any way where `data` may be a bad descriptor of the value. Can we change to `x`?